scripts/devops_tasks/test_run_samples.py

```diff
@@ -21,7 +21,12 @@
 root_dir = os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "..", ".."))
 
 IGNORED_SAMPLES = {
-    "azure-eventgrid": ["__init__.py"]
+    "azure-eventgrid": [
+        "__init__.py",
+        "consume_cloud_events_from_eventhub.py",
+        "consume_cloud_events_from_service_bus_queue.py",
+        "consume_cloud_events_from_storage_queue.py",
```

Comment:
it's not practical to test these these 4 samples - they have a dependency on storage, servicebus, EH, azure functions.
it's creating a lot of maintainance in tests.yml and in adding more dependcies to dev_requirements. Also, it's not simple to deploy azure functions on CI sample run and moreover, we do have consume samples which mimic the data we receive.

Reply:
you're farsighted, thanks for calling it out!

but I'm afraid that in the long term we want to have tests against those integration samples. Because it helps us verify if there're any changes in EH/SB/Storage SDK and we'll need to upgrade these EG samples accordingly.

I'm not sure how other SDKs is doing, but I think it would be nice to have tests for integration samples.
I didn't intent to block the PR, we could open separate issue and PRs when we have time.

thoughts? @rakshith91 @lmazuel

Reply:
that's a good point - i agree that integration tests can be useful in the long run when we eventually have some breaking changes - what i can do is add these tests once we move the test framework to use the arm template in the march milestone - that way, we have less maintenance on tests.yml

Reply:
sounds good to me!

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/16057